src/views/components/dashboard/charts/chart-slow.vue

```diff
@@ -41,6 +46,13 @@ limitations under the License. -->
     @Prop() private item!: any;
     @Prop() private type!: any;
     @Prop() private intervalTime!: any;
+    private isServiceChart: boolean = false;
+
+    private created() {
+      const serviceMetricNames = ['service_cpm', 'service_resp_time', 'service_apdex', 'service_sla'];
```

Comment:
You are hard coded these, which is not allowed.

Reply:
I will make changes as soon as possible.